symfony/plugins/orangehrmHelpPlugin/lib/processor/ZendeskHelpProcessor.php

```diff
@@ -22,6 +22,7 @@ class ZendeskHelpProcessor implements HelpProcessor {
     const DEFAULT_CONTENT_TYPE = "application/json";
     const ZENDESK_SEARCH_DIRECTORY_URL = '/api/v2/help_center/articles/search.json?';
     const ZENDESK_CATEGORY_DIRECTORY_URL = '/api/v2/help_center/categories';
```

Comment:
Why we defined const as `DIRECTORY` those are API endpoints :thinking:

Reply:
removed the DIRECTORY part of the variable